src/app/courses/courses-list/course-item/course-item.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="courseItemContainer">
+    <div class="courseItemUpperBlock">
+        <p class="courseItemTitle">{{title}}</p>
+        <p class="courseItemDuration">{{duration}}</p>
+        <p class="courseItemCreationDate">{{formattedDate}}</p>
+    </div>
+    <div class="courseItemLowerBlock">
+        <p class="courseItemDescription">{{description}}</p>
+        <button class="courseEdit">Edit</button>
+        <button class="courseDelete" (click)="onDeletedCilck()">Delete</button>
```

Comment:
try to create a separate component **Button** and reuse it, it will simplify the writing styles for button
You can use _Output()_ to observe the click event on the button

Reply:
Done